src/createPublication.ts

```diff
@@ -1,14 +1,14 @@
 import { ConfigOpts } from "./config";
-import { createFile, EmptyBatchError } from "./core/batch";
-import { filterIterable, AsyncOrSyncIterable } from "./core/utilities";
-import { Publication } from "./core/contracts/publisher";
-import { getRandomString } from "./core/utilities/random";
 import {
   AnnouncementWithSignature,
   AnnouncementType,
   SignedAnnouncement,
   TypedAnnouncement,
 } from "./core/announcements";
+import { createFile, EmptyBatchError } from "./core/batch";
```

Comment:
Especially for long PRs like this one, It would be nice to leave out formatting changes like this.

Reply:
fixed